src/main/java/reposense/system/CommandRunner.java

```diff
@@ -57,9 +57,15 @@ public static void checkoutToDate(String root, String branchName, Date untilDate
         }
 
         Path rootPath = Paths.get(root);
-
         String substituteCommand = "git rev-list -1 --before="
-                + GIT_LOG_UNTIL_DATE_FORMAT.format(untilDate) + " " + branchName;
+                + GIT_LOG_UNTIL_DATE_FORMAT.format(untilDate);
+
+        if (!branchName.isEmpty()) {
+            substituteCommand += " " + branchName;
+        } else {
+            substituteCommand += " HEAD";
+        }
```

Comment:
if specifying a branch is necessary, can simply name the default branch name to be head.